event_pubsub/event_repository.py

```diff
@@ -39,19 +39,23 @@ def read_token_stake_events(self):
 
     def update_mpe_raw_events(self, processed, row_id, error_code, error_message):
         try:
+            self.connection.begin_transaction()
             update_events = 'UPDATE mpe_events_raw SET processed = %s , error_code = %s, error_msg = %s WHERE row_id = %s '
             update_events_reponse = self.connection.execute(update_events,
                                                             [processed, error_code, error_message, row_id])
+            self.connection.commit_transaction()
 
         except Exception as e:
             self.connection.rollback_transaction()
             raise e
 
     def update_registry_raw_events(self, processed, row_id, error_code, error_message):
         try:
+            self.connection.begin_transaction()
             update_events = 'UPDATE registry_events_raw SET processed = %s, error_code = %s, error_msg = %s WHERE row_id = %s '
             update_events_reponse = self.connection.execute(update_events,
                                                             [processed, error_code, error_message, row_id])
+            self.connection.commit_transaction()
 
         except Exception as e:
             logger.exception(f"Error while updating the registry_raw_event {str(e)}")
```

Comment:
Shall we have rollback of transaction in case of any exception, especially sql exception.